ecms_base/themes/custom/ecms/templates/content/node--event--full.html.twig

```diff
@@ -0,0 +1,145 @@
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+
+{# Start date variables #}
+{% set start_date = node.field_event_date.value|date('F j, Y') %}
+{% set datetime_startdate_format = node.field_event_date.start_value|date('Y-m-d') %}
+
+{# End date variables #}
+{% set end_date = node.field_event_date.end_value|date('F j, Y') %}
+{% set datetime_enddate_format = node.field_event_date.end_value|date('Y-m-d') %}
+
+{# Time variables #}
+{% set start_time = node.field_event_date.value|date('g:ia') %}
+{% set datetime_starttime_format = node.field_event_date.value|date('H:i:s') %}
+{% set end_time = node.field_event_date.end_value|date('g:ia') %}
+{% set datetime_endtime_format = node.field_event_date.end_value|date('H:i:s') %}
+
+{% if start_time != end_time %}
+  {% set time = start_time ~ ' - ' ~ end_time %}
+{% else %}
+  {% set time = start_time %}
+{% endif %}
+
+{# Check for all day #}
+{%  if datetime_starttime_format == '00:00:00' and datetime_endtime_format == '23:59:00' %}
+  {% set allday = TRUE %}
+{% endif %}
+
+{# Generate responsive image style #}
+{% set img = '' %}
+{% if node.field_event_image is not empty %}
+  {% set img_uri = node.field_event_image.entity.field_media_image_1.entity.getFileUri() %}
+  {% set img_alt = node.field_event_image.entity.field_media_image_1.alt %}
+
+  {# TODO: this fails if an uri exists in DB but not on local file system. Need to find a way to check if file exists locally #}
+  {% set img = drupal_image(img_uri, 'wide', {alt: img_alt}, true) %}
+{% endif %}
+
+{# Get virtual meeting link url #}
+{% if node.field_event_virtual_meeting_url.0.url.external %}
+  {% set virtual_link_url = content.field_event_virtual_meeting_url.0['#url'].getUri() %}
+{% else %}
+  {% set virtual_link_url = content.field_event_virtual_meeting_url.0['#url'] %}
+{% endif %}
+
+{# Get registration link url #}
+{% if node.field_event_virtual_meeting_url.0.url.external %}
+  {% set registration_link_url = content.field_event_virtual_meeting_url.0['#url'].getUri() %}
+{% else %}
+  {% set registration_link_url = content.field_event_virtual_meeting_url.0['#url'] %}
+{% endif %}
```

Comment:
May need to review / test based on translation settings update

Reply:
These are using the content render array rather than the node entity. I'm not sure we can use the same method

Reply:
Right, this isn't an entity reference. Got it,